### PR TITLE
SM-12 Added basic routes and mock of layout

### PR DESCRIPTION
**Pull request by RobertS-ki**
Edit 01.05.2022 - updated uppercase in routes.
Added routes:
obraz

### CODE REVIEW

**Review by n0macx — CHANGES_REQUESTED** [review]
Please name Your commits and Pull Requests starting with Jira code of an issue (SM- in this case).

**Review by n0macx — CHANGES_REQUESTED** [review]
Maybe it would be good to create components in separate files ?

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
 function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  return (
+    <div className="App">
+      <Routes>
+        <Route element={<Layout />}>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/aboutUs" element={<AboutUs />} />
+          <Route path="/contact" element={<ContactPage />} />
+          <Route path="/productSeries" element={<ProductSeriesPage />} />
```
Comment (vieraboschkova) [inline src/App.tsx]: avoid uppercase, urls should contain only lowercase

File: src/App.tsx [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+import UserPage from './features/userPage/UserPage';
+
 function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  return (
+    <div className="App">
+      <Routes>
+        <Route element={<Layout />}>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/aboutUs" element={<AboutUs />} />
```
Comment (vieraboschkova) [inline src/App.tsx]: about-us, products or products-series, so you dont need uppercase

Comment (RobertS-ki) [reply src/App.tsx]: fixed

**Review by vieraboschkova — APPROVED** [review]

**Review by n0macx — APPROVED** [review]

**Pull request merged**